stvisualizer/packages/vs-syntax-tree/src/representations/dropdown-tree.tsx

```diff
@@ -0,0 +1,68 @@
+"use strict";
```

Comment:
Let's use the `PascalCase` or `camelCase` for file names everywhere. [This](https://github.com/airbnb/javascript/tree/master/react#naming) is just a reference. We just need to be consistent everywhere. That'll be enough.